docs/conda_env.yml

```diff
@@ -1,11 +1,8 @@
 name: nbconvert_docs
-channels:
-- asmeurer
 dependencies:
-- pandoc
-- nbformat
-- jupyter_client
 - sphinx
+- nbconvert
```

Comment:
nbconvert shouldn't be included here, because it's installed as part of readthedocs.yml. If we install nbconvert to build the docs, it should get the latest stable version, not the current one.